README.md

```diff
@@ -104,7 +104,7 @@ If you have any other questions on using Rivet, or have any other ideas, feel fr
 
 ## Contributors ✨
 
-Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/docs/en/emoji-key)):
+Thanks go to these wonderful people ([emoji key](https://allcontributors.org/docs/en/emoji-key)):
```

Comment:
Technically both are probably fine https://english.stackexchange.com/questions/15429/which-is-correct-special-thanks-go-to-or-special-thanks-goes-to